### PR TITLE
PLT-746 Fix if statement in "promote" workflow

### PR DESCRIPTION
**Pull request by christopher-maboh**
## 🎫 Ticket

[PLT-746](https://jira.cms.gov/browse/PLT-746)

## 🛠 Changes

"fi" was added at the end of set environment variable statement

## ℹ️ Context

ab2d team is trying to deploy some code to "prod-test" and looks like there are errors with existing code syntax in the. "promote" work flow .

## 🧪 Validation

Link to a successful manually trigger
https://github.com/CMSgov/ab2d/actions/runs/11666218254

### CODE REVIEW

File: .github/workflows/promote.yml [hunk 1]
```diff
@@ -47,6 +47,7 @@ jobs:
             echo "TAG_PREFIX=ab2d-prod-test" >> $GITHUB_ENV
           else
             echo "TAG_PREFIX=ab2d-$DEPLOYMENT_ENV" >> $GITHUB_ENV
+            fi
```
Comment (mjburling) [inline .github/workflows/promote.yml:50]:
```suggestion
          fi
```

[the remark this answers is not in the document]
Comment (christopher-maboh) [reply .github/workflows/promote.yml]: fixed

Comment (christopher-maboh) [reply .github/workflows/promote.yml]: change made

**Review by gsf — APPROVED** [review]
lgtm

**Review by ildesenesence — APPROVED** [review]
Need to make sure we have ab2d-prod-test images for this to find, but this code fix looks good

**Pull request merged**